app/forms/refunds/applicant_form.rb

```diff
@@ -44,7 +44,7 @@ class ApplicantForm < Form
                               length: { maximum: EMAIL_ADDRESS_LENGTH }
     validates :applicant_address_telephone_number, presence: true
     validates :applicant_date_of_birth, presence: true
-    validates :has_name_changed, nil: true
```

Comment:
In rails 5, the Yes / No radio button group sends an empty string in the params, previously nothing was sent so the value was nil when unanswered.  
This validator is now nil OR empty string, both meaning the same thing - 'Unanswered question'